eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/tcp/client/recommend/EventMeshRecommendImpl.java

```diff
@@ -31,63 +31,82 @@
 import java.util.List;
 import java.util.Map;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import lombok.extern.slf4j.Slf4j;
 
 @Slf4j
 public class EventMeshRecommendImpl implements EventMeshRecommendStrategy {
-    
-    private final EventMeshTCPServer eventMeshTCPServer;
 
-    public EventMeshRecommendImpl(EventMeshTCPServer eventMeshTCPServer) {
+    private static final int DEFAULT_PROXY_NUM = 1;
+
+    private final transient EventMeshTCPServer eventMeshTCPServer;
+
+    public EventMeshRecommendImpl(final EventMeshTCPServer eventMeshTCPServer) {
         this.eventMeshTCPServer = eventMeshTCPServer;
     }
 
     @Override
-    public String calculateRecommendEventMesh(String group, String purpose) {
+    public String calculateRecommendEventMesh(final String group, final String purpose) throws Exception {
         List<EventMeshDataInfo> eventMeshDataInfoList;
-        if (StringUtils.isAnyBlank(group, purpose)) {
-            log.warn("EventMeshRecommend failed,params illegal,group:{},purpose:{}", group, purpose);
+
+        if (StringUtils.isBlank(group) || StringUtils.isBlank(purpose)) {
```

Comment:
May I know what's the difference between using `isAnyBlank` and `isBlank` here?  I think the functionality is the same.

Reply:
I have fixed it. @horoc